docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -3189,15 +3189,27 @@ Then, at the end of your documentation, include the extensive documentation:
 
 Finally, generate the documentation and check it out.
 
+[[ecosystem]]
+== Ecosystem integration
+
+Some extensions may be private, and some may wish to be part of the
+broader Quarkus ecosystem, and available for community re-use.
+Inclusion in the Quarkiverse Hub] is a convenient mechanism for handing
+continuous testing and publication.
+The link:https://github.com/quarkiverse/quarkiverse/wiki#getting-an-extension-onboarded[Quarkiverse hub wiki] has instructions
```

Comment:
Not sure if it should be capitalized but let's be consistent between above and this one.
```suggestion
The link:https://github.com/quarkiverse/quarkiverse/wiki#getting-an-extension-onboarded[Quarkiverse Hub wiki] has instructions
```